lib/ohai/plugins/root_user.rb

```diff
@@ -0,0 +1,28 @@
+#
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:RootUser) do
+  provides 'root_user'
+
+  collect_data do
+    case ::RbConfig::CONFIG['host_os']
```

Comment:
It would be more resilient to have this plugin declare a dependency on the `'os'` attribute and then `case os`. On Windows, `os` returns `'windows'` (see: https://github.com/chef/ohai/blob/master/lib/ohai/mixin/os.rb#L50).